### ISSUE TITLE
loadCSV, parse and autoType=false

### ISSUE THREAD

**Issue by ericemc3** [uwdata/arquero#184]
This with `autoType: false` does not change c values:
```
aq.fromCSV(`a,b,c\r
            00152,2,01/01/2021\r
            30219,4,01/01/2021`, 
           { autoType: false,
             parse: { a: v => v,
                      c: v => v.split('/').reverse().join('-')
                   } })
```

But this with `autoType: true` works as expected:
```
aq.fromCSV(`a,b,c\r
            00152,2,01/01/2021\r
            30219,4,01/01/2021`, 
           { autoType: true,
             parse: { a: v => v,
                      c: v => v.split('/').reverse().join('-')
                   } })
```

I don't see why first block behaves differently

**Comment by jheer** [comment]
Thanks for reporting! This is certainly a bug. Will fix shortly.

**Comment by jheer** [comment]
Fix released in 4.7.1.

**Comment by ericemc3** [comment]
Thanks a lot!
